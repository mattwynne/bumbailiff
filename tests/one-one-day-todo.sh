#!/bin/bash
source "../../../test_helpers.sh"
export GIT_AUTHOR_DATE="$(yesterday)"
export GIT_COMMITTER_DATE="$(yesterday)"
mkdir -p git && cd git
create_git_repo
git commit --quiet -m 'Initial commit' --allow-empty
echo '// TODO: 1-day old TODO' >file.js
git add .
git commit --quiet -m 'Commit'
bumbailiff
echo $?
