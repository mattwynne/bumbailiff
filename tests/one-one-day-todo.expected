[1mfile.js:1[0m
[0;32m1 days old[0;37m <a-commit-ref> (Daisy Developer)[0m
TODO: 1-day old TODO

ðŸŒ± [0;32m1 TODO-days accumulated (max allowance: 14 ðŸ’µ).[0m
0
